src/bci_build/package/appcontainers.py

```diff
@@ -542,3 +542,65 @@ def _get_nginx_kwargs(os_version: OsVersion):
     )
     for os_version in (OsVersion.TUMBLEWEED,)
 ]
+
+
+_PODMAN_FILES = {"podman-containers.conf": "", "containers.conf": ""}
+
+for fname in _PODMAN_FILES:
+    _PODMAN_FILES[fname] = (Path(__file__).parent / "podman" / fname).read_text()
+
+_PODMAN_VERS_REGEX = "%%podman_version%%"
+
+PODMAN_CONTAINERS = [
```

Comment:
please move it into a podman.py so that we follow the "one-container-per-file" paradigm that I started some time ago (but fuzzed over on appcontainers, which is now becoming a mess again)